lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -84,6 +84,7 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * @returns {Promise.<AuthenticationResult>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
      */
     async acquireTokenSilent(request: SilentRequest): Promise<AuthenticationResult> {
+        const endMeasurement = this.performanceManager.startMeasurement("acquireTokenSilent");
```

Comment:
Let's create an enum for the measurement names, so that when we expose an api, apps can use that enum to filter which values they get back.

Reply:
Agree. And keep it crisp: START, END, FLUSH which is standard across performance metrics.